stdpopsim/catalog/HomSap/__init__.py

```diff
@@ -174,6 +174,36 @@
     )
 _species.add_genetic_map(_gm)
 
+for pop in ["ACB", "ASW", "BEB", "CDX", "CEU", "CHB", "CHS", "CLM", "ESN",
+            "FIN", "GBR", "GIH", "GWD", "IBS", "ITU", "JPT", "KHV", "LWK",
+            "MSL", "MXL", "PEL", "PJL", "PUR", "STU", "TSI", "YRI"]:
+    _gm = stdpopsim.GeneticMap(
+        species=_species,
+        id="Pyrho{}_GRCh37".format(pop),
+        description="Pyrho population-specific map for {}".format(pop),
+        long_description="""
+        This genetic map was inferred using individuals from the {}
+        population from Phase 3 of the 1000 Genomes Project. Rates
+        were estimated using pyrho (https://github.com/popgenmethods/pyrho)
+        while using population-specific population size history estimates
+        obtained from smc++ (https://github.com/popgenmethods/smcpp).
+        Genetic maps are only available for the 22 autosomes.
+        See https://doi.org/10.1126/sciadv.aaw9206 for more
+        details.""".format(pop),
+        url=(
+            "https://stdpopsim.s3-us-west-2.amazonaws.com/genetic_maps/"
+            "HomSap/Pyrho{}_GRCh37.tar.gz".format(pop)),
+        file_pattern=(
+            "Pyrho{}_GRCh37_chr{{id}}.txt".format(pop)),
+        citations=[
+            stdpopsim.Citation(
```

Comment:
maybe pull this citation out of here and set it to `_spence_song_1993` as with the other citations?